clouds/azure/session_clients.py

```diff
@@ -0,0 +1,29 @@
+from azure.identity import ClientSecretCredential
```

Comment:
Please add this to ,pre-commit-config.yaml

```- repo: https://github.com/pre-commit/mirrors-mypy
    rev: v1.9.0
    hooks:
      - id: mypy
        exclude: ^(tests/)
        include: <this file path>
        additional_dependencies: [types-all]
```

Reply:
done